db/migrate/20200426165114_add_schedulable_to_nodes.rb

```diff
@@ -0,0 +1,5 @@
+class AddSchedulableToNodes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :nodes, :schedulable, :boolean, null: false, default: true
```

Comment:
I think we should reduce/avoid default value in db by as much as possible

Reply:
Done. Please check.